font/src/directwrite/mod.rs

```diff
@@ -143,11 +143,17 @@ impl crate::Rasterize for DirectWriteRasterizer {
             bidiLevel: 0,
         };
 
+        let rendering_mode = font.get_recommended_rendering_mode_default_params(
+            glyph.size.as_f32_pts(),
+            self.device_pixel_ratio * (96.0 / 72.0),
+            dwrote::DWRITE_MEASURING_MODE_NATURAL
```

Comment:
Just out of curiosity, do you know anything about the measuring mode by chance? Is this maybe something we can also ask Windows about rather than setting it ourselves?

Reply:
The measuring mode is left entirely up to the application, there's no real default because it's down to how you're trying to use DirectWrite.

It's basically a switch between how DirectWrite (Natural) and GDI works, the intent was for applications replacing their existing text layout to be able to keep their existing UI metrics (via GDI compatible layout), while also allowing resolution independent layout for things like eBook/PDF readers, browser engines, etc.

But that should only have an effect if you're doing "flowing" text layout, Alacritty is rendering single glyph buffers to a fixed grid which should completely negate any differences.

Reply:
Thanks a lot for the detail, good to know!